packages/app/src/runs/RunsEmpty.vue

```diff
@@ -10,7 +10,10 @@
     <p class="h-48px mb-8px text-gray-600">
       {{ t("runs.empty.description") }}
     </p>
-    <RecordPromptAdapter />
+
+    <span class="w-full">
+      <RecordPromptAdapter />
+    </span>
```

Comment:
Wrapper with `w-full` className was added here to maintain existing UI, with fully spanned TerminalPrompt.

Reply:
```suggestion
    <RecordPromptAdapter class="w-full"/>
```
I see why you had to add the additional `w-full` class here since this component is inside a `flex` container with the direction set to column and the align-items set to center.  `span` would not be appropriate here semantically, however you do not need it at all.  It should work to add a class prop to the `RecordPromptAdapter` and Vue will pass it down to add it to the list of classes on the top level element that is rendered.  https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes.

After this update, I think this PR will be good!

Reply:
Thanks @warrensplayer that works.